Publish docker images to ghcr.io failed

### describe your issue

Error response from daemon: Get "https://ghcr.io/v2/": denied: denied
Error: Process completed with exit code 1.

Ref: https://github.com/commit-check/commit-check/actions/runs/16115136198/job/45467211353#step:4:72

### commit-check version

_No response_

### .commit-check.yml

```yaml

```

--- a/.github/workflows/publish-image.yml
+++ b/.github/workflows/publish-image.yml
@@ -2,6 +2,7 @@ name: publish image
 
 permissions:
   contents: read
+  packages: write
 
 on:
   push:

